src/map/utils/petutils.h

```diff
@@ -61,6 +61,136 @@ enum PETID
     MAX_PETID                = 77,
 };
 
+struct Pet_t
+{
+    uint16      PetID;     // ID in pet_list.sql
+    look_t      look;      // внешний вид
+    std::string name;      // имя
+    ECOSYSTEM   EcoSystem; // эко-система
+
+    uint8 minLevel; // минимально-возможный  уровень
+    uint8 maxLevel; // максимально-возможный уровень
+
+    uint8  name_prefix;
+    uint8  radius; // Model Radius - affects melee range etc.
+    uint16 m_Family;
+    uint32 time; // время существования (будет использоваться для задания длительности статус эффекта)
```

Comment:
`// lifetime (will be used to set the duration of the status effect)`  
is what it says there. They only maybe useful non-english comment that was here. Lets just delete the other non-english ones.

Reply:
updated